services/net/image/ImageAction.cs

```diff
@@ -100,7 +100,6 @@ public ImageAction(IApiService api, IOptions<ImageOptions> options, ILogger<Imag
                 {
                     // If another process has it in progress only attempt to do an import if it's
                     // more than an 5 minutes old. Assumption is that it is stuck.
-                    reference = await UpdateContentReferenceAsync(reference, WorkflowStatus.InProgress);
                 }
```

Comment:
I don't understand what you are doing here?  The reason this code is here is to act as a locking mechanism so that other processes don't attempt to work with the same content at the same time.

Reply:
Updated, initially thought this could be redundant as we just retrieve the reference on line 96